stdlib/std.jsonnet

```diff
@@ -483,23 +483,36 @@ limitations under the License.
     local pad_right(str, w, s) =
       str + padding(w - std.length(str), s);
 
-    // Render an integer (e.g., decimal or octal).
-    local render_int(n__, min_chars, min_digits, blank, sign, radix, zero_prefix) =
-      local n_ = std.abs(n__);
-      local aux(n) =
-        if n == 0 then
-          zero_prefix
+    // Render a sign & magnitude integer (radix ranges from decimal to binary).
+    // neg should be a boolean, and when true indicates that we should render a negative number.
+    // mag must always be a whole number >= 0, it's the magnitude of the integer to render
+    // min_chars must be a whole number >= 0
+    //   It is the field width, i.e. std.length() of the result should be >= min_chars
+    // min_digits must be a whole number >= 0. It's the number of zeroes to pad with.
+    // blank must be a boolean, if true a postive number has a ' ' in place of the '-'.
```

Comment:
The detailed explanation is good, but I think these lines are confusing.
* "blank must be a boolean, if true a postive number has a ' ' in place of the '-'." <- It's weird to say that a positive number has something in place of '-'. A positive number has no '-'. Perhaps it would be better to say that it adds an additional ' ' in front of the number, so that it is aligned with negative numbers with the same number of digits.
* plus must be a boolean, if true a '+' is used in place of the '-'. <- Same, but even more confusing. Naively, it sounds like we would format `-5` as `"+5"` when `plus` is true.